### PR TITLE
[DEV APPROVED] 9121 impact principles page

### PR DESCRIPTION
**Pull request by davidtrussler**
[TP9121](https://moneyadviceservice.tpondemand.com/RestUI/Board.aspx#page=board/5624876525867731357&appConfig=eyJhY2lkIjoiREI4ODcwRjkxMDNDRTM2NTlBMzhDNTRBRTVBNUU1N0UifQ==&searchPopup=userstory/9121)

Adds a new static page: Impact Principles (screenshot below). 

Additionally makes 2 changes to the style-guide necessitated by the introduction of this page: 
1. Updates the lettered list to remove the link
1. Adds a new bullet list

image

### CODE REVIEW

**Comment by masbuild** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false"><table>
  <tbody>
    <tr>
      <td>app/controllers/static_pages_controller.rb</td>
      <td>3</td>
      <td>Layout/EmptyLineBetweenDefs: Use empty lines between method definitions.</td>
    </tr>
  </tbody>
</table>
</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There are code changes, but no corresponding tests. Please include tests if this PR introduces any modifications in behavior. If this is a refactor PR, please ignore the message.Refer to our mas standards guides about opening a PR<a href="link-to-the-mas-standards-about-opening-a-pr">here</a></td>
    </tr>
  </tbody>
</table>

At Money Advice Service, we use Danger gem, to ensure that our
commits and pull requests follow  [standards](https://github.com/moneyadviceservice/mas-standards/blob/master/Dangerfile).
If you think one of the points here has been raised in error (or   shouldn't be in our standards), open an issue or pull request on [our   Danger config](https://github.com/moneyadviceservice/mas-standards).


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: app/assets/stylesheets/components/_list.scss [hunk 1]
```diff
@@ -11,6 +11,26 @@
   }
 }
 
+.list--bullets {
```
[the remark this answers is not in the document]
Comment (davidtrussler) [reply app/assets/stylesheets/components/_list.scss]: I've removed the class from this page and removed the commit to add the component to the style-guide.

**Comment by davidtrussler** [comment]
@rhianwatkins1 I've addressed the points you have raised.

**Comment by rhianwatkins1** [comment]
Perfect

**Pull request merged**